docs/content/TextInput.mdx

```diff
@@ -103,31 +167,53 @@ TextInput is a form component to add default styling to the native text input.
     name="block"
     type="boolean"
     defaultValue="false"
-    description={
-      <>
-        Creates a full width input element
-      </>
-    }
+    description={<>Creates a full width input element</>}
   />
   <PropsTableRow
     name="contrast"
     type="boolean"
     defaultValue="false"
     description="Changes background color to a higher contrast color"
   />
-  <PropsTableRow
-    name="monospace"
-    type="boolean"
-    defaultValue="false"
-    description="Shows input in monospace font"
-  />
   <PropsTableRow name='size' type="'small' | 'medium' | 'large'"     description="Creates a smaller or larger input than the default." />
 
+{' '}
+<PropsTableRow name="isLoading" type="boolean" description="Whether the to show a loading indicator in the input" />
```

Comment:
We talked about renaming this to `loading` in [slack](https://github.slack.com/archives/C02NUUQ9C30/p1647018114746609)
```suggestion
<PropsTableRow name="loading" type="boolean" description="Whether the to show a loading indicator in the input" />
```

Reply:
Yup, on it